### ISSUE TITLE
QLog 0.6.0, Settings: Modes

### ISSUE THREAD

**Issue by DL2KI** [foldynl/QLog#11]
In the tab "Modes" you find 20 modes. Obviously the classic modes "CW", "LSB", "USB", "RTTY", "DATA" are missing. All available modes are "Disabled" for me, because they are not used.

This leads to the fact that when logging a CW QSO the mode cannot be selected in the QSO window. Whether an automatic takeover of the mode from the rig into the menu fields "Mode" / "Submode cannot be checked.

**Comment by foldynl** [comment]
That's strange. Unfortunately, I cannot confirm your finding. v.0.6.0 should have 44 modes where the mentioned modes are present. See screenshot from the release. Was there the same problem observed by you with the previous version (0.5.0)?  

![modes](https://user-images.githubusercontent.com/942769/157045280-a617d366-491d-456c-8aa2-702fd0acbfe1.png)

**Comment by DL2KI** [comment]
Here is a screenshot of the window.

![settings, modes](https://user-images.githubusercontent.com/26311402/157085694-ce01275f-ffac-4277-a09b-85d4d00a5f8c.jpg)

**Comment by DL2KI** [comment]
> Have you ever tried to edit a QLog database data file?

No.

> Sorry for spamming.
> 
> I am continuing to investigate the issue but I have a workaround. The table is fully editable therefore you can redefine for example CHIP mode to CW with pre-defined Report format and DXCC should be CW/

No problem. I just tried this once and it seems to work. Tomorrow I will test this further.

According to my memory, this was also the case with version 0.5.0. I just uninstalled version 0.6.0 (purge), and reinstalled version 0.5.0. The settings menu looks the same for both versions. 
 
The database is in the user directory and of course has not changed. I will backup and delete the database tomorrow and check it again.

**Comment by DL2KI** [comment]
I have uninstalled version 0.6.0 (purge) and backed up the database "qlog.db". After that I installed version 0.5.0.

After starting QLog the database is regenerated. The new file "qlog.db" has a size of 512,0 KiB.

The Modes menu looks like this:

![v0 5 0_neueDatenbank](https://user-images.githubusercontent.com/26311402/157179394-783db552-2a87-4c02-85b1-18818baf38bd.jpg)

When the newly generated database is opened with "SQLite Studio", the content of the "modes, Indices" table looks like this:

![v0 5 0_neueDatenbank_data](https://user-images.githubusercontent.com/26311402/157179691-96f04eba-1209-46d6-b301-b05d0a999593.jpg)

I can't say more about the database, as I have no experience with SQLite or other databases. I installed "SQLite Studio" yesterday only once to have a look at the contents of the database.

After an update to version 0.6.0 nothing changes in the new database.
